src/components/Favorites.tsx

```diff
@@ -1,8 +1,44 @@
+import { useState, useEffect } from 'react';
+import api from '../services/api';
+import classNames from 'classnames';
+
+import ContactCard from './ContactCard';
+import DropDown from './DropDown';
+import View from './View';
+
+import '../styles/Favorites.scss';
+
 const Favorites = () => {
+    const [favoriteContacts, setFavoriteContacts] = useState<any[] | null>(null);
+    const [sortValue, setSortValue] = useState<number | null>(null);
```

Comment:
sortValue Is not used anywhere. It seems user won’t know which sorting did he choose. Is it possible to reset sorting?